internal/services/databricks/parse/databricks_virtual_network_peering.go

```diff
@@ -0,0 +1,131 @@
+package parse
```

Comment:
this file shouldn't be needed anymore, since it's available within `go-azure-sdk`?

Reply:
Fixed.